osu.Desktop/DiscordRichPresence.cs

```diff
@@ -19,15 +20,23 @@ namespace osu.Desktop
 {
     internal class DiscordRichPresence : Component
     {
-        private const string client_id = "367827983903490050";
+        public DiscordRichPresence(OsuConfigManager config)
```

Comment:
Rather than passing in the whole config, expose a bindable and perform the binding in `OsuGameDesktop`. Look at something like `DrawableRuleset.IsPaused` for an example of how this is done.